README.md

```diff
@@ -10,10 +10,10 @@ Our mission is to identify key policy and process choices related to corporate o
 
 The Steering Committee shall be elected for their expertise and contribution to the advancement of open source program management. They are responsible for overall operations and budgetary issues:
 
-* Chris Aniszczyk (@caniszczyk) - (8/15/2019 -  12/31/2021)
-* Duane O'Brien (@duaneobrien) - (8/15/2019 - 12/31/2021)
-* Stormy Peters (@stormypeters - (1/1/2021 - 12/31/2022)
-* Kevin P. Fleming (@kpfleming) (8/15/2019 - 12/31/2021)
-* Dawn Foster (@geekygirldawn) (1/1/2021- 12/31/2022)
-* Stephen Jacobs (@itprofjacobs) - Appointed, (1/1/2021 - 12/31/2022)
-* Ashley Wolf (@ashleywolf)- Appointed, (6/1/2020 - 12/31/2021)
+- [Ashley Wolf](https://github.com/ashleywolf), GitHub
+- [Dawn Foster](https://github.com/geekygirldawn), VMWare
+- [Shilla Saebi](https://github.com/shillasaebi), Comcast
+- [Stephen Augustus](https://github.com/justaugustus), Cisco
+- [Stephen Jacobs](https://github.com/itprofjacobs), RIT
+- [Thomas Steenbergen](https://github.com/tsteenbe), HERE Technologies
+- [VM (Vicky) Brasseur](https://github.com/vmbrasseur), Wipro
```

Comment:
Can we add the dates back in? It is super helpful when preparing for the next election to know when people's terms end.

Reply:
We also need to know which seats were appointed, so can you add that data back into the table?

Reply:
> Can we add the dates back in? It is super helpful when preparing for the next election to know when people's terms end.
> We also need to know which seats were appointed, so can you add that data back into the table?

Fixed! Added a table (that includes elected vs appointed) sorted by start date, then end date, then first name.

Reply:
(someone please check my math on the terms :))